### ISSUE TITLE
On the sixth step of the Marriage Certificate site, the drop down elements do not show the relevant information when hovering over them

### ISSUE THREAD

**Issue by ollie-iterators** [CityOfBoston/Iterators#155]
Screengrab/videos (insert file names here)
2020 08 25, Marriage Certificate Step 6 Drop down fields not having text in field shown.mp4
[2020 08 25, Marriage Certificate Step 6 Drop down fields not having text in field shown.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5132297/2020.08.25.Marriage.Certificate.Step.6.Drop.down.fields.not.having.text.in.field.shown.mp4.zip)

2020 08 25, Marriage Certificate Step 7 Drop down fields not having text in field shown.mp4
[2020 08 25, Marriage Certificate Step 7 Drop down fields not having text in field shown.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5132292/2020.08.25.Marriage.Certificate.Step.7.Drop.down.fields.not.having.text.in.field.shown.mp4.zip)

Link
https://registry.boston.gov/marriage

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOs 10.15.6 > Chrome 84

Trusted Tester Test
N/a

Brief Description
On the sixth step of the Marriage Certificate site, the drop down elements do not show the relevant information when hovering over them

Steps to reproduce
1)	From the site listed above, go through the steps and go to the fourth step of the Marriage Certificate site
2)	00:02 The dropdown fields should have output showing which dropdown text is selected. The text only describes the dropdown fields. This is a bug.

Desired Result
•	The dropdown fields show the relevant information that is necessary for understanding what it is used for

Actual Result
•	The dropdown fields for quantity and payment type do not show the contents of the drop down elements. Also, the button to expand the dropdown for Quantity has no accessible name, label, or title and is hidden using aria-hidden = true.

**Comment by ollie-iterators** [comment]
This also happens on Step 7 with the State/Territory drop down field

**Comment by ollie-iterators** [comment]
This happens on Step 2 and 3 of the Marriage Intention page: https://registry-certs.digital-staging.boston.gov/marriageintention

**Comment by david-iterators** [comment]
Please see the attached information for additional information on the accessibility information needed.
[DropDown Lists.pdf](https://github.com/CityOfBoston/Iterators/files/5496095/DropDown.Lists.pdf)
[DropDown Lists.zip](https://github.com/CityOfBoston/Iterators/files/5496096/DropDown.Lists.zip)
